src/define/Advert.ts

```diff
@@ -123,9 +123,21 @@ const isSizeMappingEmpty = (sizeMapping: SizeMapping): boolean => {
 	);
 };
 
+//We are grouping all fronts-banner slots and subsequent inline slots except inline1 to get the most use of prebid bid caching
+const getGroupedSlot = (advertId: string): string => {
+	if (advertId.match(/fronts-banner/)) {
```

Comment:
not sure if regex's are necessary, `includes` might be better? Like https://github.com/guardian/commercial/blob/8bb34702d16943891069ac9930adbd392116e6d4/src/define/Advert.ts#L75

Reply:
It could be includes inline and !includes inline1? But up to you

Reply:
d6867772d1f5cfc4749e6e523801485fa2c26cca 👍🏼